frontend/src/metabase/dashboard/reducers.js

```diff
@@ -242,10 +244,20 @@ const isAddParameterPopoverOpen = handleActions(
   INITIAL_DASHBOARD_STATE.isAddParameterPopoverOpen,
 );
 
+const isNavigatingToDashboard = handleActions(
```

Comment:
I find this name a bit misleading. If I understand this correctly, it is more like "navigating back to the dashboard you previously visited via the back button". But the name sounds more generic like it could even be true when you're navigating to any dashboard from, say, the side nav.

Would something like `isNavigatingBackToDashboard` be better? It emphasize that you're going **back** to the previously visited dashboard.